<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureToNSBTX</class>
 <widget class="QMainWindow" name="TextureToNSBTX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="scrollarea_verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <widget class="QPushButton" name="addTexture_pb">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>32</width>
             <height>32</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">padding-top:-4px;
color: rgb(0, 192, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="removeTexture_pb">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>0</y>
             <width>32</width>
             <height>32</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">padding-top:-4px;
color: rgb(255, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="textures_sa">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="textures_sa_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>579</width>
             <height>85</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="console_verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="nsbtxPath_le"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="selectNsbtxPath_pb">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Destination NSBTX File:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="console_tb">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="convert_pb">
          <property name="text">
           <string>Convert And Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
